frontend/src/components/resource/Resource.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import './Resource.css';
+import Farm from '../../assets/buildings/farm.svg';
+import Mine from '../../assets/buildings/mine.svg';
+import Food from '../../assets/sources/FoodIcon.svg';
+import Gold from '../../assets/sources/GoldIcon.svg';
+
+export default function (props) {
```

Comment:
@kkeszthe `({ kingdomId })`

Reply:
and use propTypes to ensure the kingdomId is in the props:
https://reactjs.org/docs/typechecking-with-proptypes.html